build.sbt

```diff
@@ -22,6 +22,8 @@ lazy val publishSettings = Seq(
   bintrayPublishIvyStyle := true
 )
 
+enablePlugins(JavaAppPackaging)
```

Comment:
А чем это будет лучше? Я просто сделал самым простым способом из доки плагина.

> On Aug 31, 2016, at 12:44 AM, Anton Kulaga <notifications@github.com> wrote:
> 
> In build.sbt <https://github.com/antonkulaga/kappa-notebook/pull/32#discussion_r76885411>:
> 
> > @@ -22,6 +22,8 @@ lazy val publishSettings = Seq(
> >    bintrayPublishIvyStyle := true
> >  )
> >  
> > +enablePlugins(JavaAppPackaging)
> А почему внизу файла на рут нельзя подвесить вместо серверапногопекейджинга?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/antonkulaga/kappa-notebook/pull/32/files/09892359ef36d7533076d45e66ee9799537586a8#r76885411>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AALGlXiEKfyiF4LZqzDQUEohJcUiMCqmks5qlKRJgaJpZM4Jw_KV>.
>